components/app_bar/app_bar.scss

```diff
@@ -58,6 +58,7 @@ $channel-view-padding: 63px;
 
             .app-bar__icon-inner,
             span:not(.pulsating_dot) {
+                // if we want to show a tourtip/pulsating dot in any of the app bar icons, these styles must be ommitted when span.pulsating_dot
```

Comment:
not necessary as part of this change, but I forgot to put this valid comment in a previous related PR.